spec/commands/xadd_spec.rb

```diff
@@ -7,11 +7,14 @@
   end
 
   before :each do
-    @redises._gsub(/\d{3}-\d/, '...-.')
+    # TODO: Redis appears to be returning a timestamp a few seconds in the future
+    # so we're ignoring the last 5 digits (time in milliseconds)
+    @redises._gsub(/\d{5}-\d/, '....-.')
   end
 
   it 'returns an id based on the timestamp' do
     t = Time.now.to_i
+    id = @redises.xadd(@key, key: 'value')
     expect(@redises.xadd(@key, key: 'value')).to match(/#{t}\d{3}-0/)
   end
```

Comment:
@jasonatball This doesn't cause spec failures for you? Inserting the extra xadd call causes the subsequent xadd to return `.........-1` rather than `.........-0` for me.

Reply:
I was maybe lucky that my machine is slow? I can see how that could happen. I'm not sure why that line got there actually. I recently did another PR and ran all the tests successfully. Probably should either delete that line or change to expect(id).